base_multi_company/tests/test_multi_company_abstract.py

```diff
@@ -280,3 +280,18 @@ def test_set_company_id(self):
         tester.company_id = False
 
         self.assertFalse(tester.sudo().company_ids)
+
+    def test_check_multiple_companies(self):
+        """Multiple companies can be checked for company consistence."""
+        companies = self.env["res.company"].create(
+            [
+                {
+                    "name": "Test multiple creation 1",
+                },
+                {
+                    "name": "Test multiple creation 2",
+                },
+            ],
+        )
+        companies._check_company()
```

Comment:
Calling directly to the involved method is not a very good test strategy, as it's prompt to fail by other reasons apart from the patch provided (or to not fail as exited early), so I wouldn't add tests this time.

Reply:
Thanks for the very fast review.
I always try to add a test when fixing something so I'm a bit sad this has to be removed, but you are making a good point: the records might have some data that would trigger an error in `_check_company` and that is allowed as long as `check_company_auto` is `False`.

Also thanks for merging already :smile: